app/helpers/provider_helper.rb

```diff
@@ -26,6 +26,18 @@ def rolled_over_and_new_nav?(provider)
     provider.rolled_over? && FeatureService.enabled?(:new_publish_navigation)
   end
 
+  def rollover_inactive_and_new_nav?
```

Comment:
Feels like these should go into their own rollover helper, they're not really specific to the provider one. Follow up card?